@kiva/kv-components/vue/KvLoanProgressGroup.vue

```diff
@@ -31,6 +31,10 @@ export default {
 			type: Number,
 			default: 0,
 		},
+		introductionCard: {
```

Comment:
I don't think we should pin these prop based variations to a specific loan card type. Can we go through and change the names in all the sub components from `introductionCard` to reflect what they are actually changing?

For example in this component I think this would be something like `useDefaultTextColor` or `useDarkText`.

I realize this may mean creating multiple props for changing differing aspects of a component but it would keep them generic and better explain the visual differences/options/variations that will become available for other new loan card designs...

Reply:
For the changes in KvLoanProgressGroup, KvLoanCallouts, KvLoanTag, we're just using the prop to add some css classes. Couldn't we just target these elements using `deep` in @kiva/kv-components/vue/KvIntroductionLoanCard.vue and add those small css classes there?